### ISSUE TITLE
OGL: Verification failed on GLRenderTargets.cpp:642

### ISSUE THREAD

**Issue by toccata10** [RPCS3/rpcs3#6152]
v0.0.6-8314, r9 fury, archlinux, mesa 19.1
Game doesn't launch: stops with:
```
E {PPU[0x1000000] Thread (main_thread) [0x013d6670]} sceNpTrophy: sceNpTrophyRegisterContext(context=0x1, handle=0x1, statusCb=*0x10e6e38, arg=*0x0, options=0x0)
F {RSX [0x01a0cc8]} RSX: St13runtime_error thrown: Verification failed (e=110): 
(in file ../rpcs3/Emu/RSX/GL/GLRenderTargets.cpp:642)
```

**Comment by AniLeo** [comment]
I assume it boots normally on Vulkan?

**Comment by toccata10** [comment]
Yes, it boots but due to radv bug, it's black screen. So, there's more Ogl feedback now ;)
Same error on:
*sega rally online arcade (npub30375)
*1942: joint stike (npub30024)

**Comment by AniLeo** [comment]
Verified on Windows 7 as well from an user with RTX 2060 trying God of War: Collection

**Comment by kd-11** [comment]
Thanks. I was looking into this today but could not find a test case to verify results. I'll push the fix soon.

**Comment by toccata10** [comment]
My 1st test (and error) was on: 3 on 3 Nhl Arcade (npub30058): I forgot to include this in my 1st post.

**Comment by kd-11** [comment]
Check affected titles with https://github.com/RPCS3/rpcs3/pull/6162

**Comment by scribam** [comment]
3 on 3 NHL® Arcade [NPEB00080] doesn't have the fatal anymore with #6162

**Comment by toccata10** [comment]
Problems solved with #6162. Nice work.

**Comment by AniLeo** [comment]
Not merged yet, reopened
